kibicara/platforms/email/model.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020 by Maike <tom@dl6tom.de>
+#
+# SPDX-License-Identifier: 0BSD
+
+from kibicara.model import Hood, Mapping
+from ormantic import Integer, ForeignKey, Model, Text
+
+
+class EmailSubscribers(Model):
+    """ This table stores all subscribers, who want to receive messages via email. """
+    id: Integer(primary_key=True) = None
+    hood: ForeignKey(Hood)
+    email: Text()
+
+    class Mapping(Mapping):
+        table_name = 'email_subscribers'
+
+
+class Email(Model):
+    """ This table is used to track the hood ID. It also stores the token secret. """
+    id: Integer(primary_key=True) = None
+    hood: ForeignKey(Hood)
+    secret: Text()
```

Comment:
Some disagree that the secret needs to be stored; but on the other hand then we don't really need this table anymore, as it doesn't really hold information.

The only reason is that a Censor object wants a database row as identity in its constructor. That's also why I thought it makes sense to add the secret here.

Reply:
Correction, we need the secret for the MDA API. everything makes sense :)